client/components/domains/registrant-extra-info/fr-form.jsx

```diff
@@ -24,17 +24,14 @@ const debug = debugFactory( 'calypso:domains:registrant-extra-info' );
 // and we don't particularly want to make the parent remember all our fields
 // so we use these values to plug missing.
 const emptyValues = {
-	countryOfBirth: '',
-	placeOfBirth: '',
-	postalCodeOfBirth: '',
 	registrantVatId: '',
 	sirenSiret: '',
 	trademarkNumber: '',
 };
 
 class RegistrantExtraInfoFrForm extends React.PureComponent {
 	static propTypes = {
-		countriesList: PropTypes.object.isRequired,
+		countriesList: PropTypes.object,
```

Comment:
Couldn't `countriesList` be removed as a whole or is there a dependency that I overlooked?